### PR TITLE
added profile

### PR DESCRIPTION
**Pull request by sachinparihar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1293"></a>

### CODE REVIEW

File: public/data/sachinparihar.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "url": "https://twitter.com/@SachinSarkar001",
+            "icon": "twitter"
+        },
+        {
+            "name": "Follow me on Instagram",
+            "url": "https://www.instagram.com/sachin_parihar001/",
+            "icon": "instagram"
+        }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/sachinparihar.json:22]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/sachinparihar.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Sachin Parihar",
+    "type": "personal",
+    "bio": "I am Sachin Parihar.Currently, I am pursuing my graduation in Computer Science from GJU College.I am a Open Source Enthusiast! & Also, interested in DevOps!",
```
Comment (kabir0x23) [inline public/data/sachinparihar.json:4]:
```suggestion
    "bio": "I am Sachin Parihar.Currently, I am pursuing my graduation in Computer Science from GJU College. I am an Open Source Enthusiast! & Also, interested in DevOps!",
```

**Review by kabir0x23 — APPROVED** [review]
Looks good 🌠

👇 If you like to correct this (Optional), + blank line at the end

File: public/data/sachinparihar.json (hunk 2 above)
Comment (sachinparihar) [reply public/data/sachinparihar.json]: > Looks good 🌠
> 
> 👇 If you like to correct this (Optional), + blank line at the end

done!

**Review by kumarsonsoff3 — APPROVED** [review]
looks Good to me 👍 Thanks for your contribution! 🚀

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**